Assets/Plugins/Source/Editor/EditorWindows/EOSSettingsWindow.cs

```diff
@@ -178,6 +216,9 @@ private void Save()
             // Save the product config editor
             _productConfigEditor.Save();
 
+            // reload the product config editor
+            await _productConfigEditor.LoadAsync();
```

Comment:
This is an `await` inside a non async method.

Reply:
I resolved this issue - marking conversation as resolved.